src/faq-user.md

```diff
@@ -20,37 +20,32 @@ While there is a delay before you can access your previously delegated tokens, t
 
 ## Is there slashing? Can I lose my delegated tokens?
 
-Currently, slashing is not enabled on the Casper MainNet. If a validator behaves poorly on the network they may be evicted from the auction and you will not earn rewards during the period that the validator is evicted. When slashing is enabled, tokens will be removed as penalty for poor/malicious behavior in the network. In that case, you will lose tokens delegated to that validator. 
+Currently, slashing is not enabled on the Casper Mainnet. If a validator behaves poorly on the network, they may be evicted from the auction and you will not earn rewards during the period that the validator is evicted. When slashing is enabled, tokens will be removed as penalty for poor/malicious behavior in the network. In that case, you will lose tokens delegated to that validator.
```

Comment:
Rather than "when slashing is enabled' (which implies it may be imminent), say "if slashing is enabled" and clarify that there are currently no plans to enable slashing. And if that ever were to happen, it would be the result of a governance vote.

Reply:
Done. :heavy_check_mark: